library/opusencoder/src/main/cpp/codec/CodecOggOpus.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
```

Comment:
You should keep the original copyright.

Also, maybe create a doc to explain where does this code come from?

Reply:
Since I wrote this file I guess that's the original copyright? 
Docs are a good idea, thanks.

Reply:
Ah, yes, sorry. If you are the author, this is the correct copyright.